client/src/components/publications/Publications.js

```diff
@@ -1,54 +1,100 @@
+/**
+ * The Publications component displays a list of publications 
+ */
+
 import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux'
 import { useParams } from 'react-router-dom'
 import { getPublicationsByTeamId } from '../../actions/publications'
-import { Button, Modal, InputGroup, FormControl } from 'react-bootstrap';
-import PublicationForm from './PublicationForm'
+import { Button, Modal, InputGroup, FormControl, Dropdown, Container, Col, Row } from 'react-bootstrap';
+import PublicationForm from './form/PublicationForm'
 import { BsFillPersonFill, BsArrowUpDown } from 'react-icons/bs'
 import { VscAdd } from 'react-icons/vsc'
 import { IconContext } from "react-icons"
-import Publication from './publication/Publication'
 import './publications.css'
-
+import AllPublications from './publicationsLayout/AllPublications';
```

Comment:
curious: Does `AllPublications` semantically mean the same thing as `Publications`? I know they are different components, but I'm worried about the naming here.

Reply:
May be I can change the componetn to LayoutAllPublication? would it be clear?

Reply:
Sure